readthedocs/api/v2/mixins.py

```diff
@@ -37,10 +37,10 @@ def _get_cache_tags(self):
             version = self._get_version()
             tags = [
                 project.slug,
-                f'{project.slug}-{version.slug}',
+                f'{project.slug}:{version.slug}',
```

Comment:
Also, we should be creating the cache-tag in a helper function instead of multiple places: here and El Proxito. This will avoid modifying multiple places in the code as this PR does.